subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r40/PluginApplicationBuildProgressCrossVersionSpec.groovy

```diff
@@ -24,7 +24,7 @@ import org.gradle.tooling.BuildException
 import org.gradle.tooling.ProjectConnection
 
 @ToolingApiVersion(">=2.5")
-@TargetGradleVersion(">=4.0")
+@TargetGradleVersion(">=4.0 <=4.1")
 class PluginApplicationBuildProgressCrossVersionSpec extends ToolingApiSpecification {
```

Comment:
What changed so that these tests no longer apply? It looks like we duplicated a lot of test cases.

Reply:
Build operation names for plugin applications changed.